ocs_ci/framework/conf/default_config.yaml

```diff
@@ -86,6 +86,7 @@ ENV_DATA:
   worker_replicas: 3
   skip_ocp_deployment: false
   skip_ocs_deployment: false
+  logging_version: '4.2'
```

Comment:
This is problematic IMO.
What will happen in 4.3+.
We probably need to fetch the version from another place

Reply:
@RazTamir When we specify some other versions here example 4.3 or 4.4 we have to pull the packages from quay.io manually to install those versions. Hence I gave just 4.2, Since OCP 4.2 is GAed no need to pull images. Fetching a version from somewhere won't be a problem, Fetching right images from quay.io and from registries for the right unreleased version will be complex.